quic-style:0.0.1

taniiicom opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. Please name your PR as `name:version` of the submitted package.

If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [x] a new package
- [ ] an update for a package

<!--
Please add a brief description of your package below and explain why you think it is useful to others. If this is an update, please briefly say what changed.
-->

Description: Explain what the package does and why it's useful.

A cohesive Typst design template that can be used in various situations—daily life, research, and experiments!  
Forked from [JamesxX/graceful-genetics](https://github.com/JamesxX/graceful-genetics) — thanks!

- https://github.com/taniiicom/typst-quic-style

Since the template received a lot of positive feedback, I thought it might be helpful to others as well, so I took the liberty of making a request.

- https://x.com/taniiicom/status/1922266766345941025
- https://x.com/taniiicom/status/1919743804191736069

<img width="300" alt="Screenshot 2025-05-14 at 17 11 24" src="https://github.com/user-attachments/assets/7db54b0b-8557-41b6-bfcb-baaa6573b51e" />
<img width="300" alt="Screenshot 2025-05-14 at 17 12 14" src="https://github.com/user-attachments/assets/146fbe94-9b37-43c8-ad1d-e740e91d2f49" />


<!--
These things need to be checked for a new submission to be merged. If you're just submitting an update, you can delete the following section.
-->

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

<!--
The following box only needs to be checked for **template** submissions. If you're submitting a package that isn't a template, you can delete the following section. See the guidelines section about licenses in the README for more details.
-->
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

Thanks for your review.

elegaanz commented on packages/preview/quic-style/0.0.1/fonts/TeX-Gyre-Heros/texgyreheros-bold.otf:
Font files can't be shipped with Typst packages for now. Can you please delete these files and add instructions on how to download and install/import these fonts in Typst to your README or in a comment in your template instead?

elegaanz reviewed (commented):
In my opinion, this package contains too many files. Only the `src`, `thumbnails` and `template` directories are actually relevant to this repository, the rest won't be accessible easily from Typst Universe and is mostly useful for the package developers. I think you should either remove all these other files, or make them accessible in Typst Universe by referencing them in your README (in the case of `docs/manual.pdf` for instance).

taniiicom commented:
@elegaanz 
Thank you for reviewing.
Sorry for the delayed update.
In commit 859b9fabfdb7b95eb63ca7a063664acb14b44a4f, I removed unnecessary files for Universe and added instructions to the README on how to install the fonts.

elegaanz reviewed (commented):
Thank you for fixing that. Two other small remarks and it should be ready to be merged.
